src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Task {
+
+    protected String at;
```

Comment:
Again, use nouns for things. Consider naming this variable as 'eventTime' or something rather than 'at'.

Reply:
Will do as well thanks!